cross/gst-plugins-bad/Makefile

```diff
@@ -0,0 +1,86 @@
+PKG_NAME = gst-plugins-bad
+PKG_VERS = 1.20.2
+PKG_EXT = tar.xz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://gstreamer.freedesktop.org/src/gst-plugins-bad
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/gstreamer cross/gst-plugins-base cross/libxml2
+
+HOMEPAGE = https://gstreamer.freedesktop.org/
+COMMENT  = GStreamer is a library for constructing graphs of media-handling components.
+LICENSE  = LGPLv2.1
+
+# Require access to glib-mkenums Python script
+ENV += PATH=$(STAGING_INSTALL_PREFIX)/bin:$$PATH
```

Comment:
When this Makefile requires glib-mkenums, I recommend to include `cross/glib` as dependency and not to trust an indirect dependency.